src/components/project/edit.vue

```diff
@@ -18,6 +18,10 @@ except according to the terms contained in the LICENSE file.
       <form @submit.prevent="submit">
         <form-group v-model.trim="name" :placeholder="$t('field.name')" required
           autocomplete="off"/>
+        <label class="form-group">
+          <markdown-textarea v-model="description" rows="10"/>
```

Comment:
I took `form-group` completely out of markdown textarea so it can be added here at the outer level. This meant I also had to add it to the submission comment component in the div that wraps the markdown textarea there.

I also made the button a lighter gray but still darker than the surrounding background, but with dark text so it doesn't look disabled. 

Didn't want to add a slot for a label, just wanted to think of it as a textarea that you can use wherever. Though it kind of expects to be nested within a `.form-group` now to get the background color of the text area to be consistent with the rest of the markdown stuff.

Reply:
two things a form label does (that are hard to get it to do in this context with the way the textarea is nested inside another div):
1. hide itself it the input doesn't have any value
2. becomes blue color when input is active